src/backupNg/backupNg.spec.js

```diff
@@ -223,6 +223,58 @@ describe("backupNg", () => {
 
       expect(vmTask.data.id).toBe(vmIdWithoutDisks);
     });
+
+    it("fails trying to run backup job with remotes without exportRetention", async () => {
+      const scheduleTempId = randomId();
+      const { id: jobId } = await xo.createTempBackupNgJob({
+        ...defaultBackupNg,
+        remotes: {
+          id: config.remotes.defaultRemote,
+        },
+        schedules: {
+          [scheduleTempId]: DEFAULT_SCHEDULE,
+        },
+        settings: {
```

Comment:
`Report when` value needs to be `never` to not interact with the plugin `Backup Reports`.